packages/plugin-search/src/types.ts

```diff
@@ -25,6 +25,7 @@ export type BeforeSync = (args: {
   payload: Payload
   req: PayloadRequest
   searchDoc: DocToSync
+  collection: string
```

Comment:
I know it comes in from the hook as `collection` but as an argument provided here I think we should name it `collectionSlug`, this is consistent with some of our other core code going forward.